cplusplus/VImage.cpp

```diff
@@ -831,37 +831,37 @@ VImage::operator()( int x, int y ) const
 }
 
 VImage
-operator+( const VImage a, const VImage b )
+operator+( const VImage& a, const VImage& b )
```

Comment:
`VImage` is just a pointer so there's no performance advantage here, and it will change the ABI.